### DISCUSSION TITLE
QR code

### DISCUSSION THREAD

**Discussion by yvesguilherme** [braully/bopepo#25] [Q&A] [answered]
Bom dia.

Há alguma implementação que coloque um qr code para pagamento via pix no boleto? especificamente nos do BB...?

**Comment by braully** [comment, marked as answer]
Bom dia Yves,

Nunca precisei para nenhum cliente, então nunca implementei. Vi que O fork do devcoffe adicionou a pouco um template do pdf com a área do qrcode para o pix. Você pode tentar usar o fork deles ou o template deles passando a informação extra para o gerador de PDF para incorporar o qrcode ao boleto.
Estou trabalhando uma branch chamado melhorias-comunidade que incorporar essa e outras melhorias de outros forks, porém sem previsão para lançamento e ainda bastante instável. Se fizer algo para a minha branch master atual, que é a versão estável, aceito PR 😁

Atenciosamente
